pimd/pim_iface.c

```diff
@@ -2066,3 +2066,24 @@ void pim_gm_interface_delete(struct interface *ifp)
 	if (!pim_ifp->pim_enable)
 		pim_if_delete(ifp);
 }
+
+
+char *pim_mod_str(enum pim_iface_mode mode, char *buf, size_t size)
```

Comment:
Why do we need to pass in a buffer here?  There is nothing dynamic at all.  Just return the strings